### ISSUE TITLE
Usability improvents 

### ISSUE THREAD

**Issue by PhillipSz** [jwilk/python-afl#1]
Tell the user that Cython/afl is required, and if it is not installed tell the user to do so.
Don't just give an error tell the user what to do.

**Comment by jwilk** [comment]
For the record, this what you currently get if you try to build python-afl without Cython installed:
```pytb
Traceback (most recent call last):
  File "setup.py", line 33, in <module>
    import Cython.Build
ImportError: No module named Cython.Build
```
And this is what you currently get if you run `py-afl-fuzz` without AFL proper installed:
```
/usr/local/bin/py-afl-fuzz: 13: exec: afl-fuzz: not found
```

In both cases, it's obvious to me what's wrong and how to fix it. But I'm probably biased.

But I've just realized that these prerequisites are not documented anywhere. I'll update the documentation shortly.

**Comment by jwilk** [comment]
In ff514a62139e230219319d75d41b7880a7749995, I documented build-time and runtime dependencies .
In ff1cdaa3ac91d073e06bc5781609c6f3f632b330, I made the error message when `afl-fuzz` is missing more user-friendly.
In 53f23eb7e181192b1af802b88fe7a7d417cbe83f, I modified `setup.py` to catch `ImportError`.

Are you satisfied with these changes?

**Comment by PhillipSz** [comment]
yeah, thanks a lot.

**Comment by jwilk** [comment]
I've released python-afl 0.5.4, which includes the aforementioned changes.
